<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saldoWindow</class>
 <widget class="QWidget" name="saldoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saldo</string>
  </property>
  <widget class="QLabel" name="labelSaldo">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>561</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="suljeButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>440</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>SULJE</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAsiakas">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>591</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTapahtuma">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>160</y>
     <width>301</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="timer">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>20</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
